### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by helloprimihub**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](https://github.com/primihub/community#development-workflow) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] Please provide a readable PR title.

### Description
Please describe what have you done in this pull request.

### CODE REVIEW

**Comment by helloprimihub** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -1,54 +1,92 @@
...
+I20230616 13:40:10.683745    28 cli.cc:598] SubmitTask time cost(ms): 1419
+# 查看结果
+cat data/result/psi_result.csv
+"intersection_row"
+X3
+...
+```
 
-## 开发者
+<p align="center"><img src="https://raw.githubusercontent.com/helloprimihub/img/main/readme/kt.gif" width=700 alt="PSI"></p>
```
Comment (fuxingbit) [inline README.md:64]: 图不要放在个人的仓库下

File: README.md [hunk 2]
```diff
@@ -1,54 +1,92 @@
-# PrimiHub
+<p align="center">
+  <img src="https://raw.githubusercontent.com/helloprimihub/img/main/readme/header1.jpeg" alt="Header">
```
Comment (xuefeng-xu) [inline README.md:2]: 还有这里

File: README.md [hunk 3]
```diff
@@ -1,54 +1,92 @@
...
+
+问题/帮助/BUG
+------------
+
+如果您在使用过程中遇到任何问题，需要我们的帮助可以[点击](https://github.com/primihub/primihub/issues/new/choose)反馈问题。
+
+欢迎添加我们的微信助手，加入「PrimiHub 开源社区」微信群。“零距离”接触**项目核心开发、密码学专家、隐私计算行业大咖**，获得更及时的回复和隐私计算的第一手资讯。
+
+<p align="center">
+  <img src="https://raw.githubusercontent.com/helloprimihub/img/main/readme/wechat.jpeg" alt="Header">
```
Comment (xuefeng-xu) [inline README.md:85]: 这里

**Review by fuxingbit — APPROVED** [review]

**Review by xuefeng-xu — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,54 +1,92 @@
...
+
+问题/帮助/BUG
+------------
+
+如果您在使用过程中遇到任何问题，需要我们的帮助可以[点击](https://github.com/primihub/primihub/issues/new/choose)反馈问题。
+
+欢迎添加我们的微信助手，加入「PrimiHub 开源社区」微信群。“零距离”接触**项目核心开发、密码学专家、隐私计算行业大咖**，获得更及时的回复和隐私计算的第一手资讯。
+
+<p align="center">
+  <img src="https://raw.githubusercontent.com/primihub/primihub/develop/doc/wechat.jpeg" alt="Header">
```
Comment (fuxingbit) [inline README.md:85]:
```
<p align="center">
  <img src="./doc/wechat.jpeg" alt="Header">
</p>
```
src可以这样改么？同步到gitee和gitcode后，访问GitHub网络不通可能会有显示问题

**Pull request merged**